src/renderers/shared/shared/__tests__/ReactUpdates-test.js

```diff
@@ -648,16 +648,18 @@ describe('ReactUpdates', () => {
           'Inner-render-1-0',
           'Inner-didUpdate-1-0',
         'Outer-didUpdate-1',
+           // Happens in a batch, so don't re-render yet
           'Inner-setState-1',
-            'Inner-render-1-1',
```

Comment:
Why did this rerender immediately before? I thought this change would only move some setState callbacks earlier.